docs/_ug/commandSummary/ItemCommands.md

```diff
@@ -9,24 +9,34 @@
+{% capture find %}{% include_relative _ug/commandSummary/itemCommandsExample/find.md %}{% endcapture %}
+{% capture sort %}{% include_relative _ug/commandSummary/itemCommandsExample/sort.md %}{% endcapture %}
+{% capture view %}{% include_relative _ug/commandSummary/itemCommandsExample/view.md %}{% endcapture %}
+{% capture inc %}{% include_relative _ug/commandSummary/itemCommandsExample/inc.md %}{% endcapture %}
+{% capture dec %}{% include_relative _ug/commandSummary/itemCommandsExample/dec.md %}{% endcapture %}
+{% capture set %}{% include_relative _ug/commandSummary/itemCommandsExample/edit.md %}{% endcapture %}
+{% capture del %}{% include_relative _ug/commandSummary/itemCommandsExample/del.md %}{% endcapture %}
 <!-- markdownlint-restore -->
 
 <!-- ===== CREATE TABLE FORMATTING IN NORMAL+ MARKDOWN ===== -->
 <!-- WE USE :variable: FOR VALUES THAT ARE TO BE SUBSTITUTED -->
 {% capture TABLE %}
-| Action                                                            | Format |
-|-------------------------------------------------------------------|--------|
-| Create a new item                                                 | :new:  |
-| List all items                                                    | :list: |
-| Search for an item                                                | :find: |
-| Sort an item by name, quantity, type, bought date or expiry date. | :sort: |
-| View information about an item                                    | :view: |
-| Increase the quantity of an item                                  | :inc:  |
-| Decrease the quantity of an item                                  | :dec:  |
-| Update the information of an item                                 | :set:  |
-| Delete an item                                                    | :del:  |
+| Action                                                            | Format | Example       |
+|-------------------------------------------------------------------|--------|---------------|
+| Create a new item                                                 | :new:  | :newexample:  |
+| List all items                                                    | :list: | :listexample: |
+| Search for an item                                                | :find: | :findexample: |
+| Sort an item by name, quantity, type, bought date or expiry date. | :sort: | :sortexample: |
+| View information about an item                                    | :view: | :viewexample: |
+| Increase the quantity of an item                                  | :inc:  | :incexample:  |
+| Decrease the quantity of an item                                  | :dec:  | :decexample:  |
+| Update the information of an item                                 | :edit: | :editexample: |
+| Delete an item                                                    | :del:  | :delexample:  |
 {% endcapture %}
 
 <!-- ===== RENDER THE ACTUAL TABLE ===== -->
```

Comment:
This file is still missing the necessary replacements to render the last column.

Reply:
Will fix this in a future PR as I can't push to this branch directly.

Reply:
Alternatively, can you add an empty last line to this file @Ferusel ? So that I can fix in this PR directly.

Reply:
Will do